dev/com.ibm.websphere.appserver.features/visibility/public/jakartaee-9.0/io.openliberty.jakartaee-9.0.feature

```diff
@@ -23,6 +23,7 @@ Subsystem-Name: Jakarta EE Platform 9.0
  io.openliberty.messagingSecurity-3.0,\
  io.openliberty.messagingServer-3.0,\
  io.openliberty.webProfile-9.0,\
- io.openliberty.xmlBinding-3.0
+ io.openliberty.xmlBinding-3.0,\
+ io.openliberty.xmlWS-3.0
```

Comment:
Not sure this should stay, but I added it to manual test that the feature played well with the other Jakarta EE features on server start-up.